pkg/netpol/connlist/connlist.go

```diff
@@ -304,7 +304,7 @@ func updatePolicyEngineWithK8sBasicObjects(pe *eval.PolicyEngine, clientset *kub
 // ConnlistFromK8sCluster returns the allowed connections list from k8s cluster resources, and list of all peers names
 // Deprecated
 func (ca *ConnlistAnalyzer) ConnlistFromK8sCluster(clientset *kubernetes.Clientset) ([]Peer2PeerConnection, []Peer, error) {
-	pe := eval.NewPolicyEngineWithOptions(ca.exposureAnalysis, ca.explain)
+	pe := eval.NewPolicyEngineWithOptions(ca.exposureAnalysis)
```

Comment:
how does this support explanation from live cluster then?

Reply:
this func is deprecated and should not be used; it was replaced by `ConnlistFromK8sClusterWithPolicyAPI` which send the explanation flag to the policy-engine